.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Synchronize the kata-containers installation with in the host filesystem.
+#
+set -x
+
+install_status=1
+
+SRC="${PWD}/_out/build_install"
+
+# Expect the host filesystem to be mounted in following path.
+HOST_MOUNT=/host
+
+# Some files are copied over /usr which on Red Hat CoreOS (rhcos) is mounted
+# read-only by default. So re-mount it as read-write, otherwise files won't
+# get copied.
+mount -o remount,rw $HOST_MOUNT/usr
+
+# The host's '/opt' and '/usr/local' are symlinks to, respectively, '/var/opt'
+# and '/var/usrlocal'. Adjust the installation files accordingly.
```

Comment:
typo: usrlocal  -> usr/local

Reply:
Actually not.
```
[core@worker-0-0 /]$ ls -lha var/
...
drwxr-xr-x. 12 root root  126 Aug 10 20:23 usrlocal
...
```

That's the structure used by Red Hat CoreOS, which is used by OpenShift.